Source/Details/ASPINRemoteImageDownloader.mm

```diff
@@ -131,7 +133,6 @@ + (void)setSharedImageManagerWithConfiguration:(nullable NSURLSessionConfigurati
 + (void)setSharedPreconfiguredRemoteImageManager:(PINRemoteImageManager *)preconfiguredPINRemoteImageManager
 {
   NSAssert(preconfiguredPINRemoteImageManager != nil, @"setSharedPreconfiguredRemoteImageManager requires a non-nil parameter");
-  NSAssert(sharedDownloader == nil, @"Singleton has been created and session can no longer be configured.");
```

Comment:
But removing it here means I can get away with calling this method directly, even after a singleton was created? Having said that, I don't care that much tbh.

Reply:
It's true (if you managed to create a `sharedDownloader` but without a `sharedPINRemoteImageManager`, which will assert below) you could make your way through here and miss the asserts. It's definitely wrong as-is though.